docs/examples/ssh/SSH_BITBUCKET.md

```diff
@@ -0,0 +1,24 @@
+# SSH service in Bitbucket on Kubernetes
+
+In addition to providing a service on HTTP(S), Bitbucket also allows remote Git operations over SSH connections. By default, Kubernetes Ingress controllers only work for HTTP connections, but some ingress controllers also support TCP connections.
+
+You can provide SSH access by creating a separate Kubernetes `LoadBalancer` service, but you can do this only if you don’t need your deployment to have the SSH service available on the same DNS name as the HTTP service.
```

Comment:
Should we swap the condition? It is a big _if_.

> If you don’t need your deployment to have the SSH service available on the same DNS name as the HTTP service you can provide SSH access ....

Reply:
Yeah, good call!